packages/help-center/src/components/help-center-gpt.tsx

```diff
@@ -197,7 +197,10 @@ export function HelpCenterGPT( { onResponseReceived }: Props ) {
 									) }
 								</div>
 								<GPTResponseDisclaimer>
-									{ __( "Generated by WordPress.com's Support AI", __i18n_text_domain__ ) }
+									{ __(
+										"Generated by WordPress.com's Support AI. Please be advised, that AI-generated content can be inaccurate or biased.",
```

Comment:
Typo?

```suggestion
										"Generated by WordPress.com's Support AI. Please be advised that AI-generated content can be inaccurate or biased.",
```

Reply:
I already changed this text, yeah it was a typo